types/browser-or-node/index.d.ts

```diff
@@ -0,0 +1,9 @@
+// Type definitions for browser-or-node 1.2
+// Project: https://github.com/flexdinesh/browser-or-node
+// Definitions by: Jussi Kinnula <https://github.com/jussikinnula>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.7
```

Comment:
You can omit this line from the header - these type definitions are super simple and should work all the way back into typescript's past.

Reply:
I changed the TS version to 2.8, as the tests were passing with that. You're right that it should work even with the oldest TypeScript versions.